src/core/qgscoordinatetransform.h

```diff
@@ -405,6 +405,17 @@ class CORE_EXPORT QgsCoordinateTransform
      */
     static void invalidateCache();
 
+    /**
+     * Computes an *estimated* conversion factor between source and destination units:
+     *
+     *   sourceUnits * conversionFactor = destinationUnits
+     *
+     * \param referenceExtent A reference extent based on which to perform the computation
+     *
+     * \since QGIS 3.4
+     */
+    double conversionFactor( const QgsRectangle &referenceExtent ) const;
```

Comment:
isn't the appropriate term *scale factor*?
at least in French, that's the term I learned in geodetic class.

Reply:
Thanks, sounds reasonable. I'll adjust.